### ISSUE TITLE
Timeout don't stop the TX in FM mode

### ISSUE THREAD

**Issue by ik4nzd** [g4klx/MMDVMHost#605]
Hi everyone.
On our repeater we use branch FM_Ext with active COS, with these versions:

MMDVMHost by G4KLX Version: 20200523 (compiled 03 Jun 2020, GitID # 85751e0)
Firmware: MMDVM 20200520 (D-Star / DMR / System Fusion / P25 / NXDN / POCSAG / FM) 19.2000 Mhz (Build: 08:54:45 Jun 2 2020)

Today we have found that when the timeout time in FM mode is exceeded (currently 180s), the voice is cut, but the repeater remains in transmission as long as the signal remains on the input.

We think it is better that, after passing the timeout, the repeater stops transmitting, otherwise the benefits of the timeout are not obtained.

Thank you
'73 de Adam, IK4NZD

**Comment by g4klx** [comment]
I disagree. With a multimode repeater, if the carrier drops, a use of one of the other digital modes may think that the repeater is available for use, which it isn't. Having the carrier on is therefore a good thing.

The timeout is not to stop the transmitter from transmitting, but to encourage users not to talk for too long.

**Comment by ik4nzd** [comment]
Ok Jonathan.
I worried about the life of the transistors of our equipment .... if someone forgets the PTT pressed ....HI
'73 de Adam, IK4NZD
